meet-data/nzpf/1319/entries.csv

```diff
@@ -4,7 +4,7 @@ Karen Froggatt,Raw,F,Masters 2,53.32,57,51,57.5,60,62.5,62.5,62.5,1,B
 Kane McKenzie,Raw,M,Juniors,64.28,66,19,92.5,95,-100,95,95,1,B
 Queenie Lawrence,Raw,F,Open,64.46,72,33,55,60,65,65,65,1,B
 Brian Froggatt,Raw,M,Masters 2,73.77,74,55,80,85,90,90,90,1,B
-Wahdi Moradi,Single-ply,M,Juniors,73.92,74,43,130,137.5,-142.5,137.5,137.5,1,B
+Wahdi Moradi,Single-ply,M,Open,73.92,74,43,130,137.5,-142.5,137.5,137.5,1,B
```

Comment:
That doesn't look valid...

Reply:
His age is down as 43, so we could go either way on this one.